src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Lottery {
+    private static final int MAX_POSSIBLE_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
```

Comment:
Don't forget to use access modifiers for your methods. It's a good practice to define whether a method should be public, protected, or private explicitly.